encoding/thrift/thriftrw-plugin-yarpc/main.go

```diff
@@ -72,12 +84,14 @@ func New(impl Interface, opts ...<$thrift>.RegisterOption) []<$transport>.Regist
 	h := handler{impl}
 	service := <$thrift>.Service{
 		Name: "<.Service.Name>",
-		Methods: map[string]<$thrift>.UnaryHandler{
-			<range .Service.Functions><if not .OneWay>"<.ThriftName>": <$thrift>.UnaryHandlerFunc(h.<.Name>),<end>
-		<end>},
-		OnewayMethods: map[string]<$thrift>.OnewayHandler{
-			<range .Service.Functions><if .OneWay>"<.ThriftName>": <$thrift>.OnewayHandlerFunc(h.<.Name>),<end>
-		<end>},
+			<$unaryWrapperFunc := printf "%s.%s" (import .UnaryWrapperImport) .UnaryWrapperFunc>
+			<$onewayWrapperFunc := printf "%s.%s" (import .OnewayWrapperImport) .OnewayWrapperFunc>
+			Methods: map[string]<$thrift>.UnaryHandler{
+				<range .Service.Functions><if not .OneWay>"<.ThriftName>": <$unaryWrapperFunc>(h.<.Name>),<end>
```

Comment:
This will result in unused imports if the service doesn't have any
methods.

You may want to do,

    <import .UnaryWrapperImport>.<.UnaryWrapperFunc>(h.<.Name>)

Inside the `<range>` instead.